app/models/idea.rb

```diff
@@ -3,7 +3,8 @@ class Idea < ActiveRecord::Base
   include URLNormalizer
   include Publishable
   include Likeable
-  default_scope { order(inspiration: :desc, featured: :desc, comments_count: :desc, created_at: :desc, id: :desc) }
+  default_scope { order(inspiration: :desc) }
```

Comment:
@krnjn we added Orderable, which applies those orderings as part of its default_scope so they're no longer necessary here